README.md

```diff
@@ -217,16 +253,22 @@ Note: Due to an issue with ARKit, you cannot start a Local Streem from a view us
 The above is sufficient to implement the Customer side of streems (the caller). If you want to implement the Pro side of streems (the callee) the following will get you started.
 
 ### Logging In
-Before you can start doing Pro things you'll need to login as a Pro. A user which is created with Streem can be designated as a Pro through the admin portal. Once designated, the user should login using their username and password:
+Before you can start doing Pro things you'll need to login as a Pro. A user which is created with Streem can be designated as a Pro through the admin portal. Once designated, the user should login using OpenId:
```

Comment:
Having this here is a bit confusing because you now also have the `Logging In` section at the top of the doc and this section makes it seem like `loginViaOpenId()` is the only way to login for Pro users. I would move this sample code up into the `Logging In` section at the top of the document as a third way of logging in (alongside invitations and Streem Tokens).

I would also expect most SDK integrations to use Streem Tokens and not this OpenID-based method. So I might emphasize the Streem Token approach as the recommended one unless the developer knows they need this other OpenID approach.